core/src/main/java/feast/core/model/FeatureTable.java

```diff
@@ -263,6 +266,84 @@ public void updateFromProto(
         .collect(Collectors.toSet());
   }
 
+  /**
+   * Return a boolean to indicate if FeatureTable contains all specified entities.
+   *
+   * @param entitiesFilter contain entities that should be attached to the FeatureTable
+   * @return boolean True if FeatureTable contains all entities in the entitiesFilter
+   */
+  public boolean hasAllEntities(List<String> entitiesFilter) {
```

Comment:
Comparison of lists here is error prone, since list has order. I think we should rather compare sets